api/tests/c/indigo-test-issue-269.c

```diff
@@ -0,0 +1,139 @@
+                                   "   19.9245   -7.0250    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0\n"
+                                   "   22.7223   -8.0074    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0\n"
+                                   "   21.2388  -10.1980    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0\n"
+                                   "   21.2905   -7.6590    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0\n"
+                                   "   12.6301   -6.6590    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0\n"
+                                   "   16.9603   -7.1590    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0\n"
+                                   "  1  2  1  0     0  0\n"
+                                   "  2  3  1  0     0  0\n"
+                                   "  3  4  1  0     0  0\n"
+                                   "  4  5  1  0     0  0\n"
+                                   "  5  6  1  0     0  0\n"
+                                   "  6  7  1  0     0  0\n"
+                                   "  7  8  1  0     0  0\n"
+                                   "  8  9  1  0     0  0\n"
+                                   "  9 10  1  0     0  0\n"
+                                   " 10 11  1  0     0  0\n"
+                                   " 11 12  1  0     0  0\n"
+                                   " 12 13  1  0     0  0\n"
+                                   " 13 14  1  0     0  0\n"
+                                   " 14 15  1  0     0  0\n"
+                                   " 15 16  1  0     0  0\n"
+                                   "  2 17  1  1     0  0\n"
+                                   "  3 18  1  1     0  0\n"
+                                   "  4 19  1  1     0  0\n"
+                                   "  5 20  1  1     0  0\n"
+                                   "  7 21  1  1     0  0\n"
+                                   "  8 22  1  1     0  0\n"
+                                   "  9 23  1  1     0  0\n"
+                                   " 10 24  1  1     0  0\n"
+                                   " 12 25  1  1     0  0\n"
+                                   " 13 26  1  1     0  0\n"
+                                   " 14 27  1  1     0  0\n"
+                                   " 16 28  1  0     0  0\n"
+                                   " 28 29  1  0     0  0\n"
+                                   " 29 30  1  0     0  0\n"
+                                   " 15 31  1  1     0  0\n"
+                                   " 29 32  1  1     0  0\n"
+                                   " 28 33  1  1     0  0\n"
+                                   " 16 34  1  1     0  0\n"
+                                   "  6 35  1  1     0  0\n"
+                                   " 11 36  1  1     0  0\n"
+                                   "M END\n";
+
+int test(int negative)
```

Comment:
Nevertheless, instead parametrization for test scenarios, it's better to use different functions with distinctive names. So, instead `test(int negative)`, I'd make two tests like (just a proposal): 
`testLoadGoodMoleculeSucceeds()`
`testLoadBadMoleculeFails()`
without a parameter and without branching the flow.
Yes, this increases duplicate code index, but it's inherent for tests.

Reply:
Please, don't review this file now.
This is not tests yet (I don't have data for them), this is just a dummy startup code for debugging of my code.
It will be rewritten